packages/dev/core/src/XR/features/WebXRDepthSensing.ts

```diff
@@ -4,16 +4,25 @@ import type { WebXRSessionManager } from "../webXRSessionManager";
 import { WebXRAbstractFeature } from "./WebXRAbstractFeature";
 import { Tools } from "../../Misc/tools";
 import { Texture } from "../../Materials/Textures/texture";
+import type { Observer } from "../../Misc/observable";
 import { Observable } from "../../Misc/observable";
 import type { Nullable } from "../../types";
 import { Constants } from "../../Engines/constants";
 import { WebGLHardwareTexture } from "../../Engines/WebGL/webGLHardwareTexture";
 import { InternalTexture, InternalTextureSource } from "../../Materials/Textures/internalTexture";
 import type { ThinEngine } from "../../Engines/thinEngine";
-import type { Engine } from "core/Engines/engine";
+import { MaterialPluginBase } from "core/Materials/materialPluginBase";
+import { ShaderLanguage } from "core/Materials/shaderLanguage";
+import type { Material } from "core/Materials/material";

```

Comment:
I've started doing all type only imports in a single line at the top before regular imports, e.g.
```ts
import type { Observer, Material, UniformBuffer, Camera } from "core/index";
```
To me it makes it much easier to read the code and see quickly what are the type only imports and what are the regualr imports. Not sure what you think of this pattern, but thought it was worth mentioning.